src/main/java/io/confluent/connect/jdbc/sink/dialect/PostgreSqlDialect.java

```diff
@@ -97,4 +97,18 @@ public void apply(StringBuilder builder, String col) {
     return builder.toString();
   }
 
+  @Override
+  protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      super.formatColumnValue(builder, schemaName, schemaParameters, type, value);
+    }
```

Comment:
Is `return;` missing here? I can't say for sure because there's no `default:` clause in super method's switch on `schemaName`. But if we don't expect to hit one of the `case`s then we might hit the buggie `case BOOLEAN` within the base `formatColumnValue`. Does it make sense that `default:` is missing?

If it doesn't I think a `return;` is missing here.